src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -215,13 +232,16 @@ def load_command_table(self, _):
         g.custom_command('update', 'update_lock')
 
     with self.command_group('group', resource_group_sdk, resource_type=ResourceType.MGMT_RESOURCE_RESOURCES) as g:
-        g.command('delete', 'begin_delete', supports_no_wait=True, confirmation=True)
+        g.command('delete', 'begin_delete',
+                  supports_no_wait=True, confirmation=True)
         g.show_command('show', 'get')
         g.command('exists', 'check_existence')
-        g.custom_command('list', 'list_resource_groups', table_transformer=transform_resource_group_list)
+        g.custom_command('list', 'list_resource_groups',
+                         table_transformer=transform_resource_group_list)
```

Comment:
Why did you change these formats? This will lead to too much change in this PR and make others uncertain about what the PR mainly does